app/models/world_location.rb

```diff
@@ -113,6 +113,10 @@ def self.geographical
     where(world_location_type_id: WorldLocationType.geographic.map(&:id)).ordered_by_name
   end
 
+  def worldwide_publishing_taxonomy_ab_test_content
```

Comment:
This looks like something I could use :)
Was considering some sort of helper class for the YAML business since I'm doing lots of nasty stuff to get my content out of it here https://github.com/alphagov/whitehall/pull/3250/files#diff-5886670c1b6ac5a5f3997948532b898dR62